sourcegraph/sourcegraph.proto

```diff
@@ -288,58 +289,115 @@ message RepoList {
 
 service RepoBadges {
 	// ListBadges lists the available badges for repo.
-	rpc ListBadges(RepoSpec) returns (BadgeList);
+	rpc ListBadges(RepoSpec) returns (BadgeList) {
+		option (google.api.http) = {
+			get: "/v1/repo_badges/list_badges"
```

Comment:
My 2 cents is that I like versioning via the header, too. One less thing to think about unless you really want to when creating the request URL. I don't think we'll be breaking backcompat on the API too often, either. And if we do, we can sunset the old API the same way GH did.